### DISCUSSION TITLE
Install Phoniebox after the new Linux Update

### DISCUSSION THREAD

**Discussion by mimi-ac** [MiczFlor/RPi-Jukebox-RFID#1412] [Q&A]
Hello everybody,

a few month ago I installed a phoniebox on a raspberrry pi 3b+ by following steps:

- install the old Raspian Lite
- install of all upgrades and updates
- install phoniebox (rm buster-install-*; wget https://raw.githubusercontent.com/MiczFlor/RPi-Jukebox-RFID/master/scripts/installscripts/buster-install-default.sh; chmod +x buster-install-default.sh; ./buster-install-default.sh)

Now I am not sure if I can install a new phoniebox the same way, because of

* the new "pulse audio"
* the new "Raspberry Pi OS Lite"

Should I use an older image of raspbian and do without "upgrade" and "update"?
Can I use alsa instead of pulseaudio although I use the new Rasberry Pi OS Lite?

Thankx

**Comment by smu** [comment]
Recently, I installed a new phonibox using Raspberry Pi OS Lite version 2021-01-11 and it works like a charm.

**Comment by Caliandroid** [comment]
Hello @mimi-ac ,

i did the same like @smu  with Pi OS Lite and RaspberryPi 3b+  one week ago because of a microSD card corrupt.

I transferred also the shared audiofolder afterwards from the broken sd card to restore all rfid tags and audio files without problems.

In fact Pi OS Lite can be recommended as preferred OS version for the Phoniebox to save space on the sd card and unnecessary data transfer.
